src/audio/Kconfig

```diff
@@ -483,6 +483,23 @@ config COMP_RTNR
 	  proprietary binary libSOF_RTK_MA_API.a, libSuite_rename.a, libNet.a and libPreset.a.
 	  Please contact antz0525@realtek.com for any question about the binary.
 
+config COMP_MFCC
+	bool "MFCC component"
+	default y
```

Comment:
This if for now default yes to get the full CI scan. Since it's quite big default n would be assumed later.